demo/data.ts

```diff
@@ -1,105 +1,148 @@
-export let single = [
+export const single = [
   {
     name: 'Germany',
-    value: 46268
+    region: 'Europe and Central Asia',
```

Comment:
I would like to optionally use it for color or shape.  Looking at https://www.gapminder.org/tools/#_chart-type=bubbles